sparse/src/KokkosSparse_par_ilut_handle.hpp

```diff
@@ -24,6 +24,12 @@
 namespace KokkosSparse {
 namespace Experimental {
 
+/**
+ * Handle for par_ilut. Contains useful types, par_ilut configuration settings,
+ * symbolic settings and scalar output info.
+ *
+ * For more info, see KokkosSparse_par_ilut.hpp doxygen
+ */
```

Comment:
@jgfouca: Taking a closer look, this is the doc you need convert to a doxygen-style comment for your handle docs. If you use VSCode, see https://github.com/kokkos/kokkos-kernels/pull/1768/files#diff-4818864d8e942a8c047e42981b7b1ff8975eca59cd24bc370ba7e3cb1f0ff4d5R27-R34.